ci/docker/android-builder/Dockerfile

```diff
@@ -69,7 +69,7 @@ RUN mkdir $HOME/.android && \
   # empty file to mitigate warning
   touch $HOME/.android/repositories.cfg && \
   echo "y" | sdkmanager --verbose \
-    "build-tools;29.0.2" \
+    "build-tools;29.0.3" \
```

Comment:
Did it fail because of that?

Reply:
No, it were flaky tests

Reply:
Ok. It wasn't clear for what I needed to look for it that pr